src/Library/demos/Toggle Button/main.blp

```diff
@@ -0,0 +1,82 @@
+using Gtk 4.0;
+using Adw 1;
+
+Adw.StatusPage {
+  title: "Toggle Button";
+  description: _("Represent active-state visually");
+
+  Box {
+    orientation: vertical;
+
+    Box {
+      orientation: horizontal;
+      halign: center;
+
+      ToggleButton button_first {
+        active: true;
+        icon-name: "audio-volume-muted-symbolic";
+        halign: center;
+        margin-end: 12;
+        margin-bottom: 18;
+      }
+
+      ToggleButton button_second {
+        active: false;
+        icon-name: "list-compact";
+        halign: center;
+        group: button_first;
+        margin-bottom: 18;
+      }
```

Comment:
ha it's a group then you should use "linked" on the parent box and no space between

please also add a label to explain what each example is about, here "Group"

For the icons - maybe best to use something with a clear on/off state like "eye-not-looking-symbolic" and "eye-open-negative-filled-symbolic"

If you move `margin-bottom: 18;` to the parent - it doesn't need to be repeated